pkg/stanza/fileconsumer/internal/reader/reader.go

```diff
@@ -337,6 +337,21 @@ func (r *Reader) GetFileName() string {
 	return r.fileName
 }
 
+func (r *Reader) GetOffset() int64 {
+	return r.Offset
+}
+
+func (r *Reader) GetFileSize() (int64, error) {
+	if r.file == nil {
+		return 0, nil
```

Comment:
I'm not sure about returning 0 if there is an error... it looks like the file is size 0. Maybe an error should be returned